packages/cocoon_common/lib/src/rpc_model/tree_status_change.dart

```diff
@@ -0,0 +1,45 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:json_annotation/json_annotation.dart';
+import 'package:meta/meta.dart';
+
+import 'base.dart';
+
+part 'tree_status_change.g.dart';
+
+@JsonSerializable(checked: true)
+@immutable
+final class TreeStatusChange extends Model {
```

Comment:
Same as the firebase TreeStatusChange?

Reply:
Yeah that's intentional - the frontend uses this as well.